src/context/reducers/groupChat.js

```diff
@@ -2,19 +2,19 @@ import {SET_ASSET_GROUP_CHAT, SET_PARTICIPANTS_GROUP_CHAT} from '../Types';
 
 const groupChatState = {
   asset: [],
-  participants: {},
+  participants: []
 };
 const groupChatReducer = (state = groupChatState, action) => {
   switch (action.type) {
     case SET_ASSET_GROUP_CHAT:
       return {
         ...state,
-        asset: action.payload,
+        asset: action.payload
       };
     case SET_PARTICIPANTS_GROUP_CHAT:
       return {
         ...state,
-        participants: action.payload,
+        participants: action.payload
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `participants` field in the `groupChatState` has been changed from an object to an array. This could potentially break any code that expects `participants` to be an object. Please ensure that all references to `participants` are updated to handle an array, or consider keeping it as an object if that makes more sense for your use case.


<!-- This is an auto-generated comment by OSS CodeRabbit -->